boards/sodaq-sara-sff/board.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2019 Kees Bakker
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-sara-sff
+ * @{
+ *
+ * @file
+ * @brief       Board common implementations for the SODAQ SARA SFF board
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ * @}
+ */
+
+#include "cpu.h"
+#include "board.h"
+#include "periph/gpio.h"
+
+void board_init(void)
+{
+    /* Disable power to the SARA/UBlox by default */
+    SARA_ENABLE_OFF;
+    gpio_init(SARA_ENABLE_PIN, GPIO_OUT);
+
+#if 0
+    /* The R4XX module has a PWR_ON pin. Make it low to
+     * switch on the module.
+     */
+    SARA_R4XX_PWR_ON_ON;
+    gpio_init(SARA_R4XX_PWR_ON_PIN, GPIO_OUT);
+#endif
```

Comment:
Under which conditions would this be enabled?

Reply:
I think I'll remove it. There are three pins to control the Ublox modem. And these pins should be controlled from a driver (which does not exist yet).